__tests__/app.test.2.js

```diff
@@ -0,0 +1,33 @@
+const pool = require('../lib/utils/pool');
```

Comment:
the convention is to have a separate test suite per controller, so thing1.test.js and thing2.test.js rather than multiple app.test.js